test_e2e.sh

```diff
@@ -115,6 +115,14 @@ function docker_kill() {
     docker kill "$1" >/dev/null
 }
 
+function docker_signal_process() {
+    if [[ -z "$3" ]]; then
+        echo "usage: $0 <id> <signal> <name>"
+        return 1
+    fi
+    docker exec -t $1 "killall -s $2 $3"
```

Comment:
`docker kill -s <signal>` exists for this (a rather hidden feature)

`docker kill -s "$2" "$1" >/dev/null`

Reply:
Thanks! Killall was not working anyway (not included in the image).